action.sh

```diff
@@ -34,6 +34,7 @@ export GIT_COMMIT_AUTHOR="${GIT_COMMITTER_NAME} <${GIT_COMMITTER_EMAIL}>"
 export ROOT_OPTIONS="${INPUT_ROOT_OPTIONS:="-v"}"
 export PRERELEASE="${INPUT_PRERELEASE:="false"}"
 export COMMIT="${INPUT_COMMIT:="false"}"
+export TAG="${INPUT_TAG:="false"}"
```

Comment:
This should be true by default:
https://python-semantic-release.readthedocs.io/en/latest/commands.html#tag-no-tag

And the line above as well is incorrect per action.yml spec.

Reply:
I think I'm going to just remove all bash expansion defaults as defaults are handled within PSR itself rather than trying to maintain these environment variables from GitHub Actions in line with the defaults of the project. Well then we also put defaults in the `action.yml` too.